README.md

```diff
@@ -292,27 +292,31 @@ function UiComponent({ name, age }) {
     )
 }
 
-export default () => {
-    // you may extract these two lines into a custom hook
-    const store = useContext(StoreContext)
-    const data = userSelector(store)
-    return UiComponent({...data})
-    // perhaps wrap it inside observer in here?
-    return observer(UiComponent({...data}))
-}
+export default () => (
+    () => {
```

Comment:
Well, this piece is correct actually :) It's just a regular component with hooks, it shouldn't be returning a function.